### PR TITLE
update snapshot docs for concurrent simulator support

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

**Comment by fastlane-bot-helper** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please provide a changelog summary in the Pull Request description @ohayon</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Review by KrauseFx — APPROVED** [review]
:+1:

File: snapshot/README.md [hunk 1]
```diff
@@ -174,6 +174,10 @@ fastlane snapshot
 
 Your screenshots will be stored in the `./screenshots/` folder by default (or `./fastlane/screenshots` if you're using [fastlane](https://fastlane.tools))
 
+New with Xcode 9, *snapshot* can run multiple simulators concurrently. This is the default behavior in order to take your screenshots as quickly as possible. This can be disabled to run each device, one at a time, by setting the `:concurrent_simulators` option to `false`.
+
+**Note:** While running *snapshot* with Xcode 9, the simulators will not be visibly spawned. So, while you wont see the simulators running your tests, they will, in fact, be taking your screenshots.
```
Comment (KrauseFx) [inline snapshot/README.md]: Is that something we also tell the user about in the snapshot output?

Comment (ohayon) [reply snapshot/README.md]: I dont think we do mention that in the output. We probably should though! 👍

Comment (KrauseFx) [reply snapshot/README.md]: Cool, is that something you're gonna submit a follow-up PR for?

Comment (dayjer) [reply snapshot/README.md]: Agreed, I was confused by this today.

**Pull request merged**